examples/nextjs/src/utils/dotcmsClient.js

```diff
@@ -0,0 +1,33 @@
+import { dotcmsClient } from "@dotcms/client";
+
+// Client for content fetching
+export const client = dotcmsClient.init({
+    dotcmsUrl: process.env.NEXT_PUBLIC_DOTCMS_HOST,
+    authToken: process.env.NEXT_PUBLIC_DOTCMS_AUTH_TOKEN,
```

Comment:
The last time we talked about this we said that we dont want to expose the token to the client side. That's why we made another client when we did the example for the content API. 

@fmontes should we share the client for both server/client side or should we go back to the approach we had before?

Reply:
In the last talk we had, we decided that the example was going to use a read-only token so it should be ok.